tests/ftests/systemd.py

```diff
@@ -27,3 +29,91 @@ def is_delegated(config, scope_name):
                 # until we figure out something better :(
                 return True
             raise re
+
+    # This function builds the absolute path of the cgroup, using the default
+    # systemd path and checks if the cgroup exists. The function can build
+    # path for all three cgroup setup modes, including empty cgroup v2.
+    @staticmethod
+    def cgroup_exists(config, ctrl_name, cgroup_name, ignore_systemd=False):
```

Comment:
Also, would it make more sense to have this function in the Cgroup class, and it could just be called `Cgroup.exists()`?

Reply:
Something like this with `get_controller_mount_point()` support for `None`
```
+    @staticmethod
+    def exists(config, ctrl_name, cgroup_name, ignore_systemd=False):
+        ctrl_mnt = Cgroup.get_controller_mount_point(ctrl_name)
+
+        if (ignore_systemd):
+            cgrp_path = os.path.join(ctrl_mnt, cgroup_name)
+        else:
+            try:
+                cmd = [ 'cat', '/run/libcgroup/systemd']
+                if config.args.container:
+                    slice_scope_path = config.container.run(cmd, shell_bool=True)
+                else:
+                    slice_scope_path = Run.run(cmd, shell_bool=True)
+            except RunError:
+                    return False
+
+            cgrp_path = os.path.join(ctrl_mnt, slice_scope_path, cgroup_name)
+
+        proc_file =  os.path.join(cgrp_path, 'cgroup.procs')
+        cmd = ['cat', proc_file]
+        if config.args.container:
+            try:
+                exists = config.container.run(cmd, shell_bool=True)
+            except RunError:
+                return False
+        else:
+            try:
+                exists = Run.run(cmd, shell_bool=True)
+            except RunError:
+                return False
+
+        return True
```